src/python-frontend/python_converter.h

```diff
@@ -47,4 +49,8 @@ class python_converter
   std::string current_func_name;
   std::string current_class_name;
   exprt *ref_instance;
+  bool is_converting_lhs = false;
+
+  // Map object to list of instance attributes
+  std::map<std::string, std::vector<std::string>> instance_attr_map;
```

Comment:
perhaps using unordered_map for better time complexity if the order doesn't matter?

Reply:
Thanks @kunjsong01. Just changed the type to unordered_map.